src/webserver/WebServer.jl

```diff
@@ -68,12 +68,32 @@ This will start the static HTTP server and a WebSocket server. The server runs _
 Pluto notebooks can be started from the main menu in the web browser.
 """
 function run(; kwargs...)
-    options = Configuration.from_flat_kwargs(; kwargs...)
-    run(options)
+    notebook = get(kwargs, :notebook, nothing)
+
+    # for backward compatibility - Configuation can only contain a single notebook for startup
```

Comment:
Why not allow the notebook config to be a vector? Then we don't need to rewrite here

Reply:
done